### ISSUE TITLE
npm build fails

### ISSUE THREAD

**Issue by javenu** [microsoftgraph/react-officeuifabric-sample#3]
Project runs fine in the local dev server, fails when trying to build using "npm run build" 
Looks like a Graph perm issue. Are additional steps required to complete a build? Thanks

![image](https://cloud.githubusercontent.com/assets/26263645/23684903/31644a42-0356-11e7-8dc9-d8286518daea.png)

[npm-debug.txt](https://github.com/microsoftgraph/react-officeuifabric-sample/files/826294/npm-debug.txt)

**Comment by DianeD** [comment]
Hi @javenu . It appears that Uglify doesn't handle the `class` instances or other ES6 features used by the Microsoft Graph client library for JavaScript (SDK). 

This should be resolved when the SDK has ES5 support (looks like it might be included in the next TypeScript version) or when the react-scripts project eventually switches from Uglify to babili. 

Here are a couple of workarounds for now:

  - Try an experimental SDK ES5 branch created by @dan-silver .  This might be a temporary workaround if you're okay with the risk of working with an experimental branch.
   https://github.com/microsoftgraph/msgraph-sdk-javascript/pull/29

  - Use a down-level compiler to transform the `class` and other ES6 features. I used [Babel](http://babeljs.io/) with the presets `es2015` and `react` to transpile the three SDK files in the *node_modules/msgraph-sdk-javascript/lib/src* directory that cause errors:  `GraphRequest.js`, `index.js`, and `ResponseHandler.js`. Then I was able to run `npm run build`. I tried it with both the online *Try it out* tool and babel-cli.
